go/src/koding/kites/gateway/gateway.go

```diff
@@ -0,0 +1,315 @@
+package gateway
```

Comment:
done in https://github.com/koding/koding/pull/8947/commits/32b6175acc2aa249fa3de8c4d20b0d3054123c5e